jest.config.cjs

```diff
@@ -113,7 +113,10 @@ const customJestConfig = {
   // ],
 
   // A map from regular expressions to module names or to arrays of module names that allow to stub out resources with a single module
-  // moduleNameMapper: {},
+  moduleNameMapper: {
```

Comment:
Weird, https://github.com/uuidjs/uuid/issues/451#issuecomment-1112329871 seems to imply this shouldn't be necessary. Maybe because we're using `jest.config.cjs` (i.e. a CommonJS config)?

Thanks for the comment anyway :)